core/adminserver/admin_server.go

```diff
@@ -82,17 +82,19 @@ var (
 )
 
 type server struct {
-	storage adminstorage.Storage
-	client  tpb.TrillianAdminClient
-	keygen  keys.ProtoGenerator
+	storage   adminstorage.Storage
+	logClient tpb.TrillianAdminClient
+	mapClient tpb.TrillianAdminClient
+	keygen    keys.ProtoGenerator
 }
 
 // New returns a KeyTransparencyAdminService implementation.
-func New(storage adminstorage.Storage, client tpb.TrillianAdminClient, keygen keys.ProtoGenerator) gpb.KeyTransparencyAdminServiceServer {
+func New(storage adminstorage.Storage, logClient, mapClient tpb.TrillianAdminClient, keygen keys.ProtoGenerator) gpb.KeyTransparencyAdminServiceServer {
 	return &server{
-		storage: storage,
-		client:  client,
-		keygen:  keygen,
+		storage:   storage,
+		logClient: logClient,
```

Comment:
I kinda think logAdmin and mapAdmin would be better names here, as 'logClient'/'mapClient' make me think of the primary log and map APIs and not their related admin ones.

Reply:
Good idea. Done.